Build/libHttpClient.Win32.C/libHttpClient.Win32.C.vcxproj

```diff
@@ -73,9 +73,11 @@
       <GenerateDebugInformation>DebugFull</GenerateDebugInformation>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <FullProgramDatabaseFile Condition="'$(Configuration)'=='Debug'">true</FullProgramDatabaseFile>
+      <ModuleDefinitionFile>libHttpClient.Win32.C.def</ModuleDefinitionFile>
     </Link>
     <Lib>
```

Comment:
Is this still relevant if we are building as a dll?

Reply:
I can test removing it

---
In reply to: [225740181](https://github.com/Microsoft/libHttpClient/pull/308#discussion_r225740181) [](ancestors = 225740181)

Reply:
Tested using the samples and it works without the <lib> tag

---
In reply to: [225740579](https://github.com/Microsoft/libHttpClient/pull/308#discussion_r225740579) [](ancestors = 225740579,225740181)